mne/selection.py

```diff
@@ -6,43 +6,49 @@
 
 from os import path
 
+from .io.meas_info import Info
+from . import pick_types
 from .utils import logger, verbose
-from .externals import six
 
 
 @verbose
-def read_selection(name, fname=None, verbose=None):
+def read_selection(name, fname=None, info=None, verbose=None):
     """Read channel selection from file
 
-    By default, the selections used in mne_browse_raw are supported*.
+    By default, the selections used in ``mne_browse_raw`` are supported.
     Additional selections can be added by specifying a selection file (e.g.
-    produced using mne_browse_raw) using the fname parameter.
+    produced using ``mne_browse_raw``) using the ``fname`` parameter.
 
-    The name parameter can be a string or a list of string. The returned
+    The ``name`` parameter can be a string or a list of string. The returned
     selection will be the combination of all selections in the file where
     (at least) one element in name is a substring of the selection name in
-    the file. For example, "name = ['temporal', 'Right-frontal']" will produce
-    a comination of "Left-temporal", "Right-temporal", and "Right-frontal".
+    the file. For example, ``name=['temporal', 'Right-frontal']`` will produce
+    a comination of ``'Left-temporal'``, ``'Right-temporal'``, and
```

Comment:
To be fair, people debate about the necessity of it. To me it's useful in extended lists like those with lots of words, combinations and groups of words, specific phrases or sayings, and other places. Imagine that last sentence without the Oxford comma and you'll catch my drift :)